rs/execution_environment/src/canister_manager/tests.rs

```diff
@@ -7970,3 +7971,49 @@ fn subnet_info_ingress_fails() {
             "cannot be called by a user",
         );
 }
+
+#[test]
+fn node_metrics_history_canister_call_succeeds() {
```

Comment:
There are currently those system tests:
```
use general_execution_tests::api_tests::node_metrics_history_another_subnet_succeeds;
use general_execution_tests::api_tests::node_metrics_history_ingress_query_fails;
use general_execution_tests::api_tests::node_metrics_history_ingress_update_fails;
use general_execution_tests::api_tests::node_metrics_history_non_existing_subnet_fails;
use general_execution_tests::api_tests::node_metrics_history_query_fails;
use general_execution_tests::api_tests::node_metrics_history_update_succeeds;
```

We should keep multi-subnet testcases and maybe non-existing subnet:
```
use general_execution_tests::api_tests::node_metrics_history_another_subnet_succeeds;
use general_execution_tests::api_tests::node_metrics_history_another_subnet_succeeds;
```

And the rest should be reproducible with `ExeuctionTest`:
```
use general_execution_tests::api_tests::node_metrics_history_ingress_query_fails;
use general_execution_tests::api_tests::node_metrics_history_ingress_update_fails;
use general_execution_tests::api_tests::node_metrics_history_query_fails;
use general_execution_tests::api_tests::node_metrics_history_update_succeeds;
```

Please do the migration and keep the same test names.